docs/DeveloperGuide.md

```diff
@@ -548,3 +548,23 @@ testers are expected to do more *exploratory* testing.
     1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
+1. Currently, the UUIDs for the recipes are unique. As such, when deleting a recipe, the corresponding UUID will no longer
```

Comment:
could probably change to "the UUIDs for the recipes are unique across deleted and non deleted recipes", since the actual problem is due to the code not replacing uuids for deleted recipes.